template/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "ComEtendoRxSubappBase",
```

Comment:
Is this name right? Isn't this the simplest base app?

Reply:
The name is ok.